include/RAJA/internal/IndexSetUtils.hxx

```diff
@@ -70,11 +70,11 @@ namespace RAJA
  *
  ******************************************************************************
  */
-template <typename CONTAINER_T>
-RAJA_INLINE void getIndices(CONTAINER_T& con, const IndexSet& iset)
+template <typename CONTAINER_T, typename ... SEG_TYPES>
+RAJA_INLINE void getIndices(CONTAINER_T& con, const IndexSet<SEG_TYPES ...>& iset)
 {
   CONTAINER_T tcon;
-  forall<IndexSet::ExecPolicy<seq_segit, seq_exec> >(iset, [&](Index_type idx) {
+  forall<ExecPolicy<seq_segit, seq_exec> >(iset, [&](Index_type idx) {
```

Comment:
No, because IndexSet is templated it became rather convoluted - is this something we want to revisit?